drracket/browser/private/option-snip.rkt

```diff
@@ -169,8 +169,8 @@
                  (if (or (send event button-down?)
                          (and tracking? (send event dragging?))
                          (and tracking? (send event button-up?)))
-                     (if (and (<= 0 (- (send event get-x) x))
-                              (<= 0 (- (send event get-y) y)))
+                     (if (and (> (send event get-x) x)
+                              (> (send event get-y) y))
                          (when (not hit?)
```

Comment:
This doesn't look right. If we say `(send event get-x)` is `b` then we are doing something like this:

```
(<= 0 (- b x))
 ;; add x to both sides
(<= x b)
;; (attempt to) reverse the order of the comparison
(> b x)
```

but this isn't equivalent when `b` is equal to `x`

Reply:
Fixed in jackfirth/resyntax#446.